Source Code/Ticket Management System/customer_menu.c

```diff
@@ -105,8 +110,38 @@ void viewFAQ()
 }
 
 void updateContactInfo()
-{
-	// TODO
+{	
+	system("CLS");
+	char email[512];
+	wchar_t password[50];
+	wchar_t name[50];
+	wchar_t phone_number[20];	
+	struct User* logged_user=auth_getUser(); //fetchs the user from the container
+	wprintf(L"Contact Info update\n");
+	wprintf(L"===============\n");
+	wprintf(L"enter 'no' every field you wish to not update.\n");
+	wprintf(L"Update email:\n");
+	input_char(email, 512);
+	wprintf(L"Update password:\n");
+	input_wchar(password, 50);
+	wprintf(L"Update name:\n");
+	input_wchar(name, 50);
+	wprintf(L"Update Phone number:.\n");
+	input_wchar(phone_number, 50);
+	if (!input_valid_email(email))
+		wprintf(L"invalid Email, email won't be updated:.\n");
+	else if (!strcmp(email,"no"))
+		wprintf(L"email won't be updated:.\n");
+	else
+		user_setEmail(logged_user,email);
+	if (wcscmp(password,L"no"))
+		user_setPassword(logged_user, password, false); //The initial state of a password is that it's not hashed yet.
+	if (wcscmp(name, L"no"))
+		user_setName(logged_user, name);
+	if (wcscmp(phone_number, L"no"))
+		user_setPhone(logged_user, phone_number);		
+	system("PAUSE");
+	
```

Comment:
Missing a call to `UserContainer` to update the changes to the user.

Reply:
`UserContainer` doesn't have an update function.
Write one, very similar to [`TicketContainer`](https://github.com/SCE-SWE-2018-G3/ticket-management-system/blob/master/Source%20Code/Ticket%20Management%20System/ticket_container.c#L13)